src/lib/dialog/dialog-ref.ts

```diff
@@ -14,10 +14,18 @@ export class MdDialogRef<T> {
   /** The instance of component opened into the dialog. */
   componentInstance: T;
 
+  /** Expose overlay backdrop click event */
+  backdropClicked: Observable<void>;
```

Comment:
That's what I thought first as well, but then there is `afterClosed` instead of `afterClose`, but I was not sure.

As for escape, there is `handleEscapeKey()` on `dialog-container.ts` already, so I think it's fair to expose that.